src/widgets/torrent/TorrentTile.tsx

```diff
@@ -184,7 +195,10 @@ function TorrentTile({ widget }: TorrentTileProps) {
         Cell: ({ cell, row }) => (
           <Flex>
             <Text className={useStyles().classes.noTextBreak}>
-              {(Number(cell.getValue()) * 100).toFixed(1)}%
+              {Number(cell.getValue()) === 1
+                ? Number(cell.getValue()) * 100
+                : (Number(cell.getValue()) * 100).toFixed(1)}
+              %
```

Comment:
```suggestion
              {(Number(cell.getValue()) * 100).toPrecision(3)}%
```
This will show 3 significant numbers, and automatically remove the ".0"

Reply:
I tested it on a couple downloads and it's work as expected.